okteto/install.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+set -e
+
+subdomain="$CLUSTER_NAME.k8s.civo.com"
+email="$EMAIL"
+adminToken=$ACCESS_KEY
+ingress=$(kubectl get svc -n=kube-system traefik -ojsonpath='{ .spec.clusterIP }')
+
+helm repo add okteto https://charts.okteto.com
+helm repo update
+kubectl create namespace okteto --dry-run=client -o yaml | kubectl apply -f - 
+kubectl apply -f https://charts.okteto.com/application-crds.yaml
+
+helm upgrade --install civo okteto/okteto-enterprise --namespace okteto -f https://github.com/civo/kubernetes-marketplace/blob/master/okteto/manifest.yaml --set email="$email" --set adminToken="$adminToken" --set subdomain="$subdomain" --set ingress.ip=$ingress --version 0.8.4
```

Comment:
shouldnt this be config.yaml instead of manifest.yaml?

Reply:
Yes, good catch!

Reply:
fixed!